src/logic/BrazeCards.test.ts

```diff
@@ -0,0 +1,247 @@
+import fc from 'fast-check';
+import appboy from '@braze/web-sdk-core';
+import { BrazeCard, BrazeCards } from './BrazeCards';
+import { CardSlotName, Extras } from './types';
+
+function errorHandler(error: Error, identifier: string): void {
+    console.log(identifier, error);
+}
+
+describe('braze content cards', () => {
+    describe('BrazeCards', () => {
+        describe('getCardsForProfileBadge', () => {
+            test('returns empty array if there are no content cards for this user', () => {
+                jest.spyOn(appboy, 'getCachedContentCards').mockImplementation(() => {
+                    return new appboy.ContentCards([], new Date());
+                });
+                const brazeCards = new BrazeCards(appboy, errorHandler);
+                const result = brazeCards.getCardsForProfileBadge();
```

Comment:
```suggestion

                const result = brazeCards.getCardsForProfileBadge();
                
```

Again also nitpicky and my personal preference (so feel free to ignore!) but I try to use the Arrange Act Assert pattern in these tests as much as possible (this line being the Act portion), with a newline separating each phase. I find it helps me visualise the different parts of the test.